### DISCUSSION TITLE
如何配置菜单的路由，使得页面显示同域名下其他系统的菜单对应的页面?

### DISCUSSION THREAD

**Discussion by kgloveyou** [umijs/umi#11301] [Q&A]
如何配置菜单的路由，使得页面显示同域名下其他系统的菜单对应的页面?

**Comment by fz6m** [comment]
需要给一个最小复现更清楚的描述你的场景，如果是微前端父子应用的话，可以看 examples 下面的 qiankun 示例：https://github.com/umijs/umi/tree/master/examples/qiankun-master

**Reply by kgloveyou** [reply]
没有使用微前端，两个应用都是单体应用，部署在同一个域下面。在A系统的菜单点击，打开B系统的页面。

**Reply by fz6m** [reply]
单体的话可以用 iframe ，这个不是 umi 要解决的问题，如果出现系统联动，最好的解决方案就是微前端。

如果是部分组件需要联动，可以抽取出来公用。
